WordPress/Classes/ViewRelated/Stats/JetpackLoginViewController.swift

```diff
@@ -0,0 +1,546 @@
+import Foundation
+import UIKit
+import SVProgressHUD
+import WordPressShared
+import WordPressComAnalytics
+
+/// A view controller that presents a Jetpack login form
+///
+class JetpackLoginViewController: UIViewController {
+
+    // MARK: - Constants
+
+    fileprivate let jetpackInstallRelativePath = "plugin-install.php?tab=plugin-information&plugin=jetpack"
+    fileprivate let jetpackMoreInformationURL = "https://apps.wordpress.com/support/#faq-ios-15"
+
+    // MARK: - Properties
+
+    typealias CompletionBlock = (Bool) -> Void
+    /// This completion handler closure is executed when the authentication process handled
+    /// by this VC is completed.
+    ///
+    open var completionBlock: CompletionBlock?
+
+    @IBOutlet fileprivate weak var jetpackImage: UIImageView!
+    @IBOutlet fileprivate weak var descriptionLabel: UILabel!
+    @IBOutlet fileprivate weak var usernameTextField: WPWalkthroughTextField!
+    @IBOutlet fileprivate weak var passwordTextField: WPWalkthroughTextField!
+    @IBOutlet fileprivate weak var verificationCodeTextField: WPWalkthroughTextField!
+    @IBOutlet fileprivate weak var scrollView: UIScrollView!
+    @IBOutlet fileprivate weak var signinButton: WPNUXMainButton!
+    @IBOutlet fileprivate weak var sendSMSCodeButton: UIButton!
+    @IBOutlet fileprivate weak var installJetpackButton: WPNUXMainButton!
+    @IBOutlet fileprivate weak var moreInformationButton: UIButton!
+
+    fileprivate var blog: Blog!
```

Comment:
Does it make sense to make this a constant (let) and not an explicitly unwrapped optional?